src/fides/api/schemas/connection_configuration/connection_secrets_dynamodb.py

```diff
@@ -25,6 +25,8 @@ class DynamoDBSchema(ConnectionConfigSecretsSchema):
         sensitive=True,
     )
 
+    # TODO: include an aws_assume_role_arn and more closely follow the pattern in `connection_secrets_s3`
```

Comment:
i don't want to make our changes too wide-ranging quite yet, so i've just put in a note that we should look to switch dynamoDB over to this authentication paradigm soon. dynamoDB does require a region (I think?), so i'm not sure how that changes things. but I still think that provided an option to assume a role will be the right thing to support here moving forward 👍